src/ralph/lib/tests/test_models.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import Permission
+from django.test import TestCase
+
+from ralph.assets.models.choices import ObjectModelType
```

Comment:
we've done sth similar in scrooge:
https://github.com/allegro/ralph_pricing/blob/develop/src/ralph_scrooge/tests/models.py
We've created models only for tests (notice that it was in Django 1.4 so there might be differences)